src/app/cli/src/graphql.ml

```diff
@@ -428,6 +440,242 @@ module Make (Commands : Coda_commands.Intf) = struct
                         transaction database") ]
       end
     end
+
+    module Pagination = struct
+      module Page_info = struct
+        type t = {has_previous_page: bool; has_next_page: bool}
+
+        let obj =
+          obj "PageInfo" ~fields:(fun _ ->
```

Comment:
this also is supposed to have "firstCursor" and "lastCursor" nullable fields (I think this is what you're supposed to pass to continue)